tuning/trackers/tracker_factory.py

```diff
@@ -60,10 +68,23 @@ def _register_aim_tracker():
         )
 
 
-def _is_tracker_installed(name):
-    if name == "aim":
-        return _is_aim_available
-    return False
+def _register_mlflow_tracker():
+    # pylint: disable=import-outside-toplevel
+    if _is_mlflow_available:
+        # Local
+        from .mlflow_tracker import MLflowTracker
+        from tuning.config.tracker_configs import MLflowConfig
+
+        mlflow_tracker = _get_tracker_class(MLflowTracker, MLflowConfig)
+
+        REGISTERED_TRACKERS[MLFLOW_TRACKER] = mlflow_tracker
+        logging.info("Registered mlflow tracker")
+    else:
+        logging.info(
```

Comment:
Yes..I will change it to error.

Reply:
This shouldn't be an error because it happens at code load time when we are registering all available trackers.
The actual error is thrown later if user requests this tracker. In this case we just inform user that if she intends to use the tracker it should be installed.

Reply:
then shouldtn't this be debug message?